scripts_morph/MorphScript.py

```diff
@@ -0,0 +1,345 @@
+﻿#Run this script, from the project root folder with $ python scripts_morph/MorphScript.py path_to_TurboMorphologicalTagger_executable/TurboMorphologicalTagger_executable
```

Comment:
This file does not comply with Python style guide conventions (see https://www.python.org/dev/peps/pep-0008/ for general guidelines and https://google.github.io/styleguide/pyguide.html for Google guideline). Some comments:
- the file name should be morph_script.py, not MorphScript.py (actually this name is not good either since it's not informative)
- variable names should all be lower_case_separated_with_underscores
- maximum number of columns should be 80; use "\" to break lines without messing the indentation

For now, since this is just a test script, I can accept this as-is, but I suggest that you take a look in the style-guides above for future reference in writing Python scripts.